cinema/migrations/0001_initial.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 5.1.2 on 2024-10-16 15:59
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Movie',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('title', models.CharField(max_length=255)),
+                ('description', models.TextField()),
```

Comment:
If you specify `max_length`, it's more reasonable to use `CharField` instead of `TextField`. Consider changing `description` to `models.CharField(max_length=255)` if you intend to limit the length.